Low-disk health emails active by default for Project Admins (#1350)
platformsh/platformsh-docs: src/administration/integrations/notifications.md

> **Note**
>
> Remember that you must have `admin` access to a project in order to add or modify an integration.  See [User administration roles](/administration/users.md#user-roles) for more details.

## Default low-disk email notifications

When you create a new project, Platform.sh creates a default [low-disk email notification](administration/integrations/notifications.md#low-disk-warning) for all [Project Admins](/administration/users.md#user-roles).

> **Note**

> > All projects without any kind of Health Notification set up by April 6th, 2020 get a default low-disk notification configured.

Author: @Crell could you review this extra note?

> > **Note**
> > All projects without any kind of Health Notification set up by April 6th, 2020 get a default low-disk notification configured.

Reviewer: ```suggestion
> **Note**
>
> All projects created prior to 6 April 2020 that did not have any health notifications enabled had an email notification added for admin users.
```

## Available notifications

### Low-disk warning

Platform.sh monitors disk space usage on all applications and services in your cluster.

> [Project Admins](/administration/users.md#user-roles) receive low-disk email notifications by default. 

Reviewer: ```suggestion
```

This was just said 2 lines up.  No need to repeat it again.

### Email notifications

> #### Default email notifications

Reviewer: I wouldn't put the defaults as a sub-section of emails.  For one, we try to avoid 4th level headers generally.

Rather, this should be a level 2 header, possibly even before "Available notifications", that calls out that projects get an email notification for admins upon creation.  Then defer the rest of the description of how email works to the email section, already in place.  Then if there's stuff to expand on (like the new #admins magic value), document those inline as a standard feature of whatever scope.

Author: Makes sense to me, thanks for explaining @Crell!

> When your project is new or when your project doesn't have a health notification set up, Platform.sh creates a default low-disk email notification for all [Project Admins](/administration/users.md).

Reviewer: Will we be adding admin notifications to all notification-free projects going forward, or is it just a one-time thing for existing projects?  Viz, if I create a project in 4 months and deliberately delete the notification, will it come back?

The text here as written suggests it would, which is not what I expected (and is probably unwise).

You can check out the default email notification by running `platform integration:get`.

```bash
platform integration:get
+--------------+---------------+
| Property     | Value         |
+--------------+---------------+
| id           | abcdefghijklm |
| type         | health.email  |
| role         |               |
| from_address |               |
| recipients   | - '#admins'   |
+--------------+---------------+
```

The following `recipients` values are available:

* `#admins` maps to all project admins and up.

> * `#viewers` maps to everyone with access to the project.

Reviewer: These shortcuts are a new feature.  They should be included in the main setup for email notifications, not tucked in here for the defaults.

A notification can trigger an email to be sent, from an address of your choosing to one or more addresses of your choosing.

> You can check out an email notification by running `platform integration:get`, choosing `health.email`.

Reviewer: ```suggestion
You can view an email notification by running `platform integration:get`.
```

```bash
platform integration:get
+--------------+---------------+
| Property     | Value         |
+--------------+---------------+
| id           | abcdefghijklm |
| type         | health.email  |
| role         |               |
| from_address |               |
| recipients   | - '#admins'   |
+--------------+---------------+
```

> To edit the `recipients` that receive the default email notification, run in your command line:

Reviewer: ```suggestion
To edit the `recipients` that receive the default email notification, use the `integration:update` command.
```

```bash
platform integration:update abcdefghijklm --recipients you@example.com
```

> The following `recipients` values are available:

Reviewer: ```suggestion
The `recipients` field may be any valid email address, or one of the following special values.
```
